tests/integration/test_against_token_server.py

```diff
@@ -336,10 +336,12 @@ def test_unique_email_token(
 
 
 @pytest.mark.parametrize(
-    "version",
-    [v2, v3],
+    "version, method",
+    [(v2, "get"), (v3, "get"), (v3, "options")],
```

Comment:
```suggestion
    [(v2, "GET"), (v3, "GET"), (v3, "OPTIONS")],
```

Let's use typical casing for HTTP verbs that people will expect.